Ginger/Ginger/Reports/HTMLReportTemplatePage.xaml.cs

```diff
@@ -92,9 +92,19 @@ public HTMLReportTemplatePage(HTMLReportConfiguration HTMLReportConfiguration)
             SetLoadedLogoImage();
             SetHTMLReportsConfigFieldsGridsView();
             SetHTMLReportsConfigFieldsGridsData(_HTMLReportConfiguration);
+            RadioButtonInit();
+        }
+        public void RadioButtonInit()
+        {
+            string currentValue = _HTMLReportConfiguration.ExecutionCalculationLevel;
```

Comment:
just set the correct radio btn based on _HTMLReportConfiguration.ExecutionCalculationLevel value instead using the tags approach